libsolidity/interface/CompilerStack.cpp

```diff
@@ -1206,13 +1206,93 @@ Json CompilerStack::ethdebug(Contract const& _contract, bool _runtime) const
 	solAssert(m_stackState >= AnalysisSuccessful, "Analysis was not successful.");
 	solAssert(_contract.contract);
 	solUnimplementedAssert(!isExperimentalSolidity());
-	if (_runtime)
+	Json result = {
+		{
+			"contract",
+			{
+					{"name", _contract.contract->name()},
+					{
+						"definition",
+						{{
+							"sources",
+							{{"id", static_cast<int>(sourceIndices()[_contract.contract->sourceUnitName()])}}
+						}}
+					}
+			}
+		},
+		{"environment ", _runtime ? "call" : "create"},
```

Comment:
```suggestion
		{"environment", _runtime ? "call" : "create"},
```

How does this not result in an error in tests?